.github/workflows/Build.yml

```diff
@@ -54,22 +54,22 @@ jobs:
       uses: actions/cache@v3
       id: docker-image-cache
       with:
-        path: kphp-build-env-${{matrix.os}}.tar
-        key: docker-image-cache-${{matrix.os}}-${{ hashFiles('.github/workflows/Dockerfile.*', 'tests/python/requirements.txt') }}
+        path: /tmp/docker-save-${{matrix.os}}
```

Comment:
why do you need an additional subfolder if it will eventually contain one tar file?

Reply:
there is no need, i was thinking that is more convinient way. backed to file now